ddsrecorder_participants/src/cpp/recorder/mcap/McapHandler.cpp

```diff
@@ -390,50 +353,39 @@ void McapHandler::stop(
     {
         if (!on_destruction)
         {
-            logWarning(
-                DDSRECORDER_MCAP_HANDLER,
-                "Ignoring stop command, instance already stopped.");
+            logWarning(DDSRECORDER_MCAP_HANDLER,
+                    "MCAP_STATE | Ignoring stop command, instance already stopped.");
```

Comment:
I don't really like this name, it's not giving information about the MCAP state but about the MCAPHANDLER

Reply:
The thing is the McapHandler is a concept that comes from how the code works. A user shouldn't be aware of what the McapHandler is or what it's in charge of. It should just know high level concepts, like state and write.